src/Forms/Form/Form.stories.mdx

```diff
@@ -1,336 +1,545 @@
+- Default Value: `array` of `string` - Array of values of the selected options 
+
+#### Props (Input Settings)
+* **handleRemove** - `void` Optional - Callback executed when the remove button is clicked. This function should empty the assetProperties array to go the set up stage.
+* **handleSetDocument** - `Promise<void>` Optional - Callback executed when the document icon clicked. It should fill the assetProperties array with the document properties.
+* **handleSetImage** - `Promise<void>` Required - Callback executed when the image icon clicked. It should fill the assetProperties array with the document properties.
+* **handleSetVideo** - `Promise<void>` Optional - Callback executed when the video icon is clicked. It should fill the assetProperties array with the document properties.
+* **handleSetLink** - `Promise<void>` Optional - Callback executed when the link icon is clicked. It should fill the assetProperties array with the document properties.
+* **options** - `array` of `object` Optional - List of menu options that can be executed by the component.
+* **src** - `string` Optional - If the asset contains an image, its source should be passed via this src prop.
+
+#### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: "imageVideoDocumentLink",
+				inputSettings: {
+					options: [
+						{
+							label: "Edit",
+							action: handleEdit,
+						},
+						{
+							label: "Translate",
+							action: handleTranslate,
+						},
+					],
+					handleSetImage: () => {
+						alert("handleSetImage is called");
+					},
+					handleSetDocument: () => {
+						alert("handleSetImage is called");
+					},
+					handleSetVideo: handleSetDocument: () => {
+						alert("handleSetVideo is called");
+					},
+					handleSetLink: handleSetDocument: () => {
+						alert("handleSetLink is called");
+					},
+					src: "Show image"
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+---
+### FormFieldMapCoordinates
+- `Map Coordinates` This component is helpful when the user needs browse coordinates in a map.
+- [**Playground**](http://kube.simpleview.io:10000/?path=/docs/formfields-formfieldmapcoordinates--playground)
+- Default Value: `object` - Object of the value arguments 
+
+#### Props (Input Settings)
+* **address** - `object` Optional - Address object used to set lat and lng values when using the autocoordinates feature.
+* **apiKey** - `string` Required - Google Maps API key needed to consume the Maps JavaScript API and Places API.
+* **mapPosition** - `object` Optional - Latitude and longitude object.
+
+#### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: "mapCoordinates",
+				inputSettings: {
+					apiKey: "AIzaSyArV4f-KFF86Zn9VWAu9wS4hHlG1TXxqac",

```

Comment:
Please change this apiKey to a random set of letters / numbers, this api key belongs to simpleview and shouldn't be visible to external people.